src/libs/ajax.js

```diff
@@ -892,7 +892,7 @@ const Workspaces = signal => ({
 
       getEntities: entityType => fetchRawls(`${root}/entities/${entityType}`, _.merge(authOpts(), { signal })).then(r => r.json()),
 
-      getEntitiesTsv: entityType => fetchOrchestration(`api/workspaces/${namespace}/${name}/entities/${entityType}/tsv?model=flexible`, _.mergeAll([authOpts(), { signal }])).then(r => r.text()),
+      getEntitiesTsv: entityType => fetchOrchestration(`api/workspaces/${namespace}/${name}/entities/${entityType}/tsv?model=flexible`, _.mergeAll([authOpts(), { signal }])),
```

Comment:
before this PR, `getEntitiesTsv` was unused. As of this PR, I'm using it, but  the `.then(r => r.text())` is wrong and should be removed

Reply:
I think I added this for data table versions. I kind of dislike how some Ajax methods return  [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) object and some return the response content (usually via Response.json). I think it would be nice if we were more consistent about that. It looks like you're using the blob form of the response content. Could this function return that instead of the Response object?

Reply:
I've updated so this method returns the blob.